addons/pos_self_order/static/src/pages/landing_page/landing_page.xml

```diff
@@ -15,7 +15,13 @@
                         <t t-esc="link.name"/>
                     </a>
                 </t>
-                <t t-if="selfOrder.table and selfOrder.self_order_mode === 'each'">
+                <a
+                    type="button"
+                    t-on-click="() => this.router.navigate('productList')"
+                    class="btn btn-primary py-3 my-2">
+                    View Menu
+                </a>
```

Comment:
As per the other comment on the removal of the custom link data, we should not hardcode this link. We should respect the customers' setup.